plugins/tinymce5/classes/Editor.php

```diff
@@ -146,6 +146,11 @@ protected static function config($config, $selector)
         $parsedconfig = str_replace('%STYLESHEET%',$pth['folder']['template'] . 'stylesheet.css',$parsedconfig);
         $parsedconfig = str_replace('%CMSIMPLE_ROOT%',CMSIMPLE_ROOT,$parsedconfig);
         
+        /*
+         * Use the codemirror theme configured for Codeeditor_XH if available.
+         */
+        $parsedconfig = str_replace('%CODEMIRROR_THEME%',$plugin_cf['codeeditor']['theme'],$parsedconfig);
```

Comment:
Where do we check the existence of the codeeditor plugin? `$plugin_cf['codeeditor']` might throw a warning.

Reply:
Good catch! Would `isset($plugin_cf['codeeditor'])` check this? I'll have a closer look later this evening.

Reply:
Done. :)